pyproject.toml

```
@@ -146,8 +146,10 @@ log_cli_level = "INFO"
 xfail_strict = true
 filterwarnings = [
     "error",
-    "ignore:pkg_resources.*:DeprecationWarning:",
-    "ignore:.*qiskit.__qiskit_version__.*:DeprecationWarning:qiskit:",
+    'ignore:.*qiskit.__qiskit_version__.*:DeprecationWarning:qiskit:',
+    'ignore:.*qiskit.utils.algorithm_globals.QiskitAlgorithmGlobals*:DeprecationWarning:qiskit',
+    'ignore:.*Building a flow controller with keyword arguments is going to be deprecated*:PendingDeprecationWarning:qiskit',
+    "ignore::DeprecationWarning:pkg_resources",
```

```suggestion
    "ignore:pkg_resources.*:DeprecationWarning:",
    "ignore:.*qiskit.__qiskit_version__.*:DeprecationWarning:qiskit:",
    "ignore:.*qiskit.utils.algorithm_globals.QiskitAlgorithmGlobals*:DeprecationWarning:qiskit",
    "ignore:.*Building a flow controller with keyword arguments is going to be deprecated*:PendingDeprecationWarning:qiskit",
```
I believe you could have just kept the existing ignore for the `pkg_resources` deprecation warning ;-)